### DISCUSSION TITLE
How do you play?

### DISCUSSION THREAD

**Discussion by EnderCodesLol** [Redfire75369/Fission-Simulator#67] [General]
The moment I start, I get some mine and reactor upgrades. How do I progress after that? There is no way to go on afterwards.

**Comment by redfire75369** [comment]
I'm assuming you were able to get at least some energy production. At about 1e12(i think) Energy, you'll unlock centrifuges which allow you to enrich fuel, 1e18-ish is the next wall since I have yet to properly balance centrifuges nor add new mechanics.

P.S: You can check out the discord to get more up to date help.

**Comment by redfire75369** [comment]
Update: I was looking through the code and testing and I saw a bug which didn't allow you to progress from the start, I have a feeling you experienced that, but it should be fixed now.

**Reply by EnderCodesLol** [reply]
Yes, that's what was happening. I'm liking it now so far and am close to reaching centrifuges, and I'm excited to see what those will do! Thanks for viewing my post =D
